app/src/main/java/helium314/keyboard/keyboard/internal/keyboard_parser/floris/KeyCode.kt

```diff
@@ -134,11 +134,13 @@ object KeyCode {
     // Code value representing the code is not specified.
     const val NOT_SPECIFIED =             -10008 // todo: not sure if there is need to have the "old" unspecified keyCode different, just test it and maybe merge
     const val CLIPBOARD_COPY_ALL =        -10009
-    const val PAGE_UP =                   -10010
-    const val PAGE_DOWN =                 -10011
-    const val META =                      -10012
-    const val META_LOCK =                 -10013 // to be consistent with the CTRL/ALT(/FN LOCK codes, not sure whether this will be used
-    const val TAB =                       -10014
+    const val WORD_LEFT =                 -10010
+    const val WORD_RIGHT =                -10011
+    const val PAGE_UP =                   -10012
+    const val PAGE_DOWN =                 -10013
+    const val META =                      -10014
+    const val META_LOCK =                 -10015 // to be consistent with the CTRL/ALT(/FN LOCK codes, not sure whether this will be used
+    const val TAB =                       -10016
```

Comment:
Changing key codes changes people's layouts. Unless you have a _really_ good reason, please just don't.

Reply:
Ah dear, I didn't even think of that. Thanks for pointing that out.